plugins/account_history_plugin/account_history_plugin.cpp

```diff
@@ -99,10 +266,24 @@ account_history_plugin::~account_history_plugin()
 
 void account_history_plugin::set_program_options(options_description& cli, options_description& cfg)
 {
+   cfg.add_options()
+         ("filter_on_accounts,f", bpo::value<vector<string>>()->composing(),
+          "Track only transactions whose scopes involve the listed accounts. Default is to track all transactions.")
+         ("get-transactions-time-limit", bpo::value<int>()->default_value(account_history_plugin_impl::DEFAULT_TRANSACTION_TIME_LIMIT),
+          "Limits the time to wait before returning for get_transactions")
```

Comment:
The time to wait before returning? Or the maximum time to spend processing a single call?